awscli/examples/controltower/tag-resource.rst

```diff
@@ -0,0 +1,10 @@
+**To Tag Control Tower Enabled Control**
+
+The following ``tag-resource`` example tags an AWS Control Tower enabled control::
+
+    aws controltower tag-resource \
+        --resource-arn "arn:aws:controltower:us-east-1:123456789012:enabledcontrol/2H2AWUG4SKG81855" \
+        --tags "TestTagKey=TestTagValue"
+
+
```

Comment:
Missing output section. Please provide output or add the line ``This command produces no output.``
https://aws.github.io/aws-cli/docs_styleguide.html#output